packages/perspective-viewer-d3fc/src/js/tooltip/tooltip.js

```diff
@@ -0,0 +1,67 @@
+export function tooltip(container, selection, settings) {
+    selection
```

Comment:
I appreciate that we pass `container` into the legend component, but in this case I don't like having to pass it down through `barSeries`, which shouldn't need the container.

If we add a `getContainer()` method to the template, could we do something like this?

````javascript
const container = getChartElement(selection.node()).getContainer();
````